docs/envs/MaMuJoCo/ma_multiagentswimmer.md

```diff
@@ -0,0 +1,82 @@
+---
+firstpage:
+lastpage:
+---
+ManySegmentSwimmer
+
+# Swimmer
+```{figure} figures/many_segment_swimmer.png
+    :name: many_segment_swimmer
+```
+
+
+This Environment is part of [MaMuJoCo](https://robotics.farama.org/envs/MaMuJoCo/) environments. Please read that page first for general information.
+The task is variation of [Gymansium's MuJoCo/Swimmer](https://gymnasium.farama.org/environments/mujoco/swimmer/), which instead of having 2 segments, it has configurable amount of segments.
+
+The task was first introduced by Christian A. Schroeder de Witt in ["FACMAC: Factored Multi-Agent Centralised Policy Gradients"](https://arxiv.org/abs/2003.06709)
+
+
+## Action Space
+The action spaces is depended on the partitioning
```

Comment:
Suggestion: "The shape of the action spaces depend on the partitioning. The partitioning has the following form: .."